tests/sentry/tasks/test_relocation.py

```diff
@@ -1971,6 +1968,38 @@ def test_success(
             relocation: Relocation = Relocation.objects.get(uuid=self.uuid)
             assert relocation.latest_unclaimed_emails_sent_at is not None
 
+    def test_success_ignore_manually_claimed_users(
+        self,
+        notifying_owner_mock: Mock,
+        fake_message_builder: Mock,
+    ):
+        with assume_test_silo_mode(SiloMode.CONTROL):
+            admin: User = User.objects.get(id=self.imported_users["1"], email="admin@example.com")
+            admin.is_unclaimed = False
+            admin.save()
+
+        self.mock_message_builder(fake_message_builder)
+
+        with patch.object(LostPasswordHash, "send_relocate_account_email") as mock_relocation_email:
+            notifying_users(self.uuid)
+
+            # Called once for each user imported that has not been manually claimed. Since we
+            # imported 2 users in `fresh-install.json`, but then manually claimed one at the top of
+            # this test, only one user remains.
+            assert mock_relocation_email.call_count == 1
+            email_targets = [
+                mock_relocation_email.call_args_list[0][0][0].username,
+            ]
+            assert sorted(mock_relocation_email.call_args_list[0][0][2]) == self.imported_orgs
```

Comment:
Mocks also have a few [assertion methods](https://docs.python.org/3/library/unittest.mock.html#unittest.mock.Mock.assert_called_with), you could likely use `mock_relocation_email.assert_called_with()`